src/authorization/accessCheckers/AgentAccessChecker.ts

```diff
@@ -0,0 +1,13 @@
+import { ACL } from '../../util/Vocabularies';
+import type { AccessCheckerArgs } from './AccessChecker';
+import { AccessChecker } from './AccessChecker';
+
+export class AgentAccessChecker extends AccessChecker {
+  public async canHandle(): Promise<void> {
+    // It can always handle the input
+  }
+
```

Comment:
```suggestion
  public async canHandle(): Promise<void> {
    // It can always handle the input
  }
  
```

this is the default already.

Reply:
(and idem in all below)

Reply:
Fixed